src/Microsoft.ComponentDetection.Detectors/cocoapods/PodComponentDetector.cs

```diff
@@ -49,7 +51,7 @@ protected override async Task OnFileFoundAsync(ProcessRequest processRequest, ID
         }
         catch (Exception e)
         {
-            this.Logger.LogFailedReadingFile(file.Location, e);
+            this.Logger.LogError(e, "Error parsing Podfile.lock");
```

Comment:
We lost debug information here relating to the file location since Podfile.lock could be in any subdirectory.